skbio/stats/distance/_permdisp.py

```diff
@@ -36,9 +36,9 @@ def permdisp(
     method="eigh",
     number_of_dimensions=10,
 ):
-    """Test for Homogeneity of Multivariate Groups Disperisons using Marti
-    Anderson's PERMDISP2 procedure.
+    """Test for Homogeneity of Multivariate Groups Disperisons.
 
+    Uses Marti Anderson's PERMDISP2 procedure.
```

Comment:
This sentence can go to the "Notes" section, and ideally have a citation. (Should be: Anderson, M.J. 2006. Distance-based tests for homogeneity of multivariate dispersions. Biometrics 62(1):245–253.)